migration_toolkit_for_containers/index.adoc

```diff
@@ -0,0 +1,79 @@
+[id="migration-toolkit-for-containers-overview"]
+= Migration toolkit for containers overview
+include::modules/common-attributes.adoc[]
+:context: migration-toolkit-for-containers-overview
+
+toc::[]
+
+You can migrate stateful application workloads between {product-title} 4 clusters at the granularity of a namespace by using the Migration Toolkit for Containers (MTC). To learn more about MTC see xref:../migration_toolkit_for_containers/about-mtc.adoc#about-mtc[Understanding MTC].
+
+[discrete]
+[id='mtc-overview-install-mtc']
+== MTC installation
+xref:../migration_toolkit_for_containers/installing-mtc.adoc#installing-mtc[MTC installation]: You can install the Migration Toolkit for Containers (MTC) on {product-title} 4. After installing MTC, you must configure an object storage to use as a replication repository. You must install the Migration Toolkit for Containers that is compatible with your {product-title} version by using OLM. You manually install the legacy Migration Toolkit for Containers Operator on {product-title} 4.5 and earlier versions. You can check the following items:
+
+* xref:../migration_toolkit_for_containers/installing-mtc.adoc#installing-mtc[Migration Toolkit for Containers Operator on {product-title}]
+* xref:../migration_toolkit_for_containers/installing-mtc.adoc#migration-configuring-proxies_installing-mtc[Proxies]
+* xref:../migration_toolkit_for_containers/installing-mtc.adoc#configuring-replication-repository_installing-mtc[Replication repository]
+* xref:../migration_toolkit_for_containers/installing-mtc.adoc#migration-configuring-mcg_installing-mtc[Multicloud Object Gateway]
```

Comment:
MCG is the only option for restricted environment.

Connected - AWS, Azure, GCP.

I think you should remove this line because they are all replication repositories, so they are covered by the previous bullet item.